.mergify.yml

```diff
@@ -23,3 +23,25 @@ pull_request_rules:
       queue:
         method: rebase
         name: default
+  - name: Needs landing
+    conditions:
+      - label=Needs Landing
+      - "#approved-reviews-by>=1"
+      - -draft
+      - label!=Work In Progress
+      - label!=Do Not Land
+    actions:
+      queue:
+        method: merge
```

Comment:
This was `rebase` before, is there a reason we changed it to plain `merge`? I just wanna make sure it's what we intend.

Reply:
Yeah. If we add rebase, once a PR is approved and although it passed on Bitrise and there are no conflicts, mergify will rebase an run the build again. I remember asking about this and it could generate a lot of builds .. I have thought about trying just merge...

Reply:
This makes sense. 👍